docs/application/native/guides/text-input/multi-assistant.md

```diff
@@ -0,0 +1,444 @@
+# Multi-assistant
+
+
+Multi-assistant allows the user to install and use multiple voice assistants on the same machine.
+
+The Multi-assistant framework will manage voice assistants that are built with Multi-assistant client library and will activate one among them when either wake word is detected or voice key is pressed.
+
+The main features of the Multi-assistant API include:
+
+- Retrieving information
+
+  You can [get various information](#info) from the Multi-assistant framework:
+
+  - Client state
+
+    The state is changed by functions and applied as a precondition of each API.
+
+  - Service state
+
+    The service state is changed when the user starts or stops interacting with installed voice assistants.
+
+  - Active state
+
+    Active state indicates whether the current client application is expected to serve as an active voice assistant for the user. An active voice assistant is responsible for responding to user utterance and should provide appropriate feedback.
+
+  - Audio Data
+
+    When the assistant gets activated, it will retrieve the user utterance audio data from the Multi-assistant framework in order to process user's voice command.
+
+- Updating result information
```

Comment:
@mijin85cho,

In my opinion, it would be good to provide a link from here to #Update result information. 
Please let me know your opinion. If yes, then we can modify the text to cover the link in the following content.

Reply:
Suggested changes uploaded : 8d1f0235392f175a5771159cdceaf853b812f1da

Reply:
Please see if the updated change would be sufficient.

Reply:
Yes, I too agree with various.